testing/py3-pillow-simd/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer: Marcel Esser <marcel.esser@gmail.com>
+pkgname=py3-pillow-simd
+_pkgname=Pillow-SIMD
+pkgver=6.0.0
+pkgrel=0
+pkgdesc="Python Imaging Library SIMD"
+url="https://github.com/uploadcare/pillow-simd"
+arch="x86_64"
+license="PSF"
+provides="py-pillow py3-pillow"
+depends="py3-olefile freetype jpeg libwebp tiff libpng lcms2 libjpeg-turbo zlib"
+makedepends="python3-dev py3-setuptools freetype-dev jpeg-dev libwebp-dev
+	tiff-dev libpng-dev lcms2-dev libjpeg-turbo-dev zlib-dev"
+subpackages="py3-${pkgname#py-}:_py3"
+source="https://files.pythonhosted.org/packages/b1/19/b7043190f481abb94dcdd1e69c4432432aaa73455cf1128eae39b8eb2518/Pillow-SIMD-$pkgver.post0.tar.gz"
```

Comment:
please use a simpler url like other py3- packages do

Reply:
I believe I have resolved this with 2adbbd8fe9a12735d28542865a16663851bde0fd. Please note that this PyPi package uses a "post-release" version number which is defined by PyPi conventions, but I have never actually seen before in any package. There also appear to be no examples of its use in existing Alpine packages. Its is analogous to Alpine's pkgrel incrementor. I decided to resolve this by adding an underscored variable to the APKBUILD file to represent it. Please do let me know if you want this to work according to a different convention.